src/main/java/HTTP/HTTPResponseDTO.java

```diff
@@ -0,0 +1,29 @@
+package HTTP;
+
+import org.json.JSONObject;
+
+public class HTTPResponseDTO {
+
+    public String status;
+    public JSONObject body;
```

Comment:
Instead of JSONObject I am more for the describing response fields and using Jackson library 
The benefit of it that you get field value via getters and don't need to remember the right name of the field

Reply:
done